dora/tests/testcontainers/src/test/java/alluxio/membership/MembershipManagerTest.java

```diff
@@ -401,4 +401,44 @@ public void testSameWorkerIdentityConflict() throws Exception {
     Assert.assertTrue(curWorkerInfo.isPresent());
     Assert.assertEquals(wkr2.getAddress(), curWorkerInfo.get().getAddress());
   }
+
+  @Test
+  public void testWorkerHttpServerPorts() throws Exception {
+    final MembershipManager membershipManager = getHealthyEtcdMemberMgr();
+    Configuration.set(PropertyKey.WORKER_MEMBERSHIP_MANAGER_TYPE, MembershipType.ETCD);
+    Configuration.set(PropertyKey.ETCD_ENDPOINTS, getClientEndpoints());
```

Comment:
u dont need to set the config here anymore if getHealthyEtcdMemberMgr is used

Reply:
Removed.